tensorflow/stream_executor/BUILD

```diff
@@ -25,7 +25,6 @@ cc_library(
         "platform/**/*.h",
     ]),
     linkopts = [
-        "-ldl",
```

Comment:
Have you investigated why this was put here in the first place? We want to preserve support for other platforms.